queries.sql

```diff
@@ -22,3 +22,51 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
 /*Find all animals with a weight between 10.4kg and 17.3kg 
 (including the animals with the weights that equals precisely 10.4kg or 17.3kg)*/
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3 OR weight_kg = 10.4 OR weight_kg = 17.3;
+
+/*Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made.
+Then roll back the change and verify that species columns went back to the state before tranasction. */
+BEGIN; 
+UPDATE animals set species = 'unspecified';
+ROLLBACK;
+
+/*Inside a transaction:
+Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.*/
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon%';
```

Comment:
**Changes Required**
- [ ] This query will update all animals containing `mon`. The objective of this query should be to update all animals ending with `mon`. Let us please try and fix this.